org.scintilla.SciTE.yaml

```diff
@@ -0,0 +1,61 @@
+app-id: org.scintilla.SciTE
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+command: SciTE
+
+finish-args:
+  - --socket=wayland
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --filesystem=host
```

Comment:
Why access to the whole host filesystem?

Reply:
Like Geany or any text editor, it may need to read/write files everywhere. What would you advise?

Reply:
https://github.com/flathub/org.geany.Geany/blob/master/org.geany.Geany.json also uses filesystem=host